README.md

```diff
@@ -56,7 +56,7 @@ There are some guidelines which will make applying PRs easier for us:
 + Make commits of logical units.
 + Respect the original code style: by using the same [codestyle][code-style],
   patches should only highlight the actual difference, not being disturbed by any formatting issues:
-  + Only use spaces for indentation.
```

Comment:
I guess this file is in every repo...

Reply:
Probably. Is there a master version somewhere from which this is derived?

Reply:
@khmarbaise Is there one?

Reply:
https://maven.apache.org/developers/conventions/code.html

Reply:
this file was added manually, then adjusted to every repo. Maybe would be good to add also recommendations/requirement to use spotless plugin (otherwise build would fail)

 IMHO this description is too long - easier maybe would be to add a link to generic instructions to out maven home page?